sdk/eventhubs/azure_messaging_eventhubs/src/producer/mod.rs

```diff
@@ -20,11 +12,18 @@ use crate::{
     models::{EventHubPartitionProperties, EventHubProperties},
 };
 use async_std::sync::Mutex;
-use azure_core::RetryOptions;
 use azure_core::{
     credentials::AccessToken,
     error::{Error, Result},
-    Uuid,
+    RetryOptions, Uuid,
+};
+use azure_core_amqp::{
+    cbs::{AmqpClaimsBasedSecurity, AmqpClaimsBasedSecurityApis},
+    connection::{AmqpConnection, AmqpConnectionApis, AmqpConnectionOptions},
+    management::{AmqpManagement, AmqpManagementApis},
+    sender::{AmqpSendOptions, AmqpSender, AmqpSenderApis, AmqpSenderOptions},
+    session::{AmqpSession, AmqpSessionApis, AmqpSessionOptions},
+    value::{AmqpSymbol, AmqpValue},
```

Comment:
This is an unfortunate - but not unexpected - effect. Still, I imagine rust-analyzer let you do this (mostly) automatically, right? Not the worst thing. Still probably more elegant than dealing with all the builders in the grand scope of things.

Reply:
In this case, the only items that had to be added were `AmqpSymbol` and `AmqpValue` - the rest are all from removing blank lines from the use statements and letting rustfmt have its way with the code.

And yes, it was semi-automatic (I think that in this particular case I did it manually because the rust analyzer doesn't always do the right thing).

Reply:
Btw, on balance, I think I prefer the no-builder variants over the builder variant.

The only part that was annoying is setting `.properties` because `Vec[("foo", value)]` didn't automatically get converted. And I actually suspect that I could have worked around that with an appropriate `impl From`.